src/main/java/core/basesyntax/strategy/BalanceActivityTypeHandler.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceActivityTypeHandler implements ActivityTypeHandler {
+    @Override
+    public FruitTransaction.Operation getActivityType() {
+        return FruitTransaction.Operation.BALANCE;
+    }
+
+    @Override
+    public int getSumOfOperation(int quantity) {
+        if (quantity < 0) {
+            throw new RuntimeException("Fruit quantity cannot be negative");
+        }
+        return quantity;
+    }
+}
```

Comment:
You should not return the value, do the operation in this method
```suggestion
public class BalanceOperationHandlerImpl implements OperationHandler {
    @Override
    public void handle(FruitTransaction fruitTransaction) {
        FruitStorage.FRUITS.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
    }
}
```